app/src/main/java/com/github/epfl/meili/models/Post.kt

```diff
@@ -1,34 +1,11 @@
 package com.github.epfl.meili.models
 
 import android.os.Parcelable
-import android.util.Log
-import com.google.firebase.firestore.DocumentSnapshot
 import kotlinx.parcelize.Parcelize
 
 @Parcelize
 data class Post(
-    val id: String,
-    val author: String,
-    val title: String,
-    val text: String
-) : Parcelable {
-
-    /** Creates an instances of Post from other objects */
-    companion object {
-        private const val TAG = "Post"
-
-        // Create Post from Firestore data
-        fun DocumentSnapshot.toPost(): Post? {
```

Comment:
How do we convert document snapshots to posts now?

Reply:
`documentSnapshot.toObject(Post::class.java)`